doc/examples/motion_planning_python_api/scripts/motion_planning_python_api_tutorial.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+"""
+A script to outline the fundamentals of the moveit_py motion planning API.
+"""
+import numpy as np
+import rclpy
+from rclpy.node import Node
+import sys
+import threading
+import time
+
+from geometry_msgs.msg import Pose, PoseStamped
+from moveit_msgs.msg import Constraints
+
+from moveit.planning import (
+    MoveItPy,
+    PlanRequestParameters,
+    MultiPipelinePlanRequestParameters,
+)
+from moveit.core.robot_state import RobotState
+from moveit.core.kinematic_constraints import construct_joint_constraint
+
+from ament_index_python.packages import get_package_share_directory
+from moveit_configs_utils import MoveItConfigsBuilder
+import yaml
```

Comment:
A lot of these imports are not used, so you can clean them up! Here is a screenshot from my VS Code showing the unused ones greyed out:

![image](https://user-images.githubusercontent.com/4603398/220784506-cce24039-e9d5-4073-898b-8563b77e8dd2.png)